light-curve/light_curve/light_curve_py/features/rainbow/bolometric.py

```diff
@@ -0,0 +1,173 @@
+from abc import abstractmethod
+from dataclasses import dataclass
+from typing import Dict, List
+
+import numpy as np
+
+__all__ = ["bolometric_terms", "BaseBolometricTerm", "SigmoidBolometricTerm", "BazinBolometricTerm"]
+
+
+@dataclass()
+class BaseBolometricTerm:
+    """Bolometric term for the Rainbow"""
+
+    @staticmethod
+    @abstractmethod
+    def parameter_names() -> List[str]:
+        return NotImplementedError
+
+    @staticmethod
+    @abstractmethod
+    def parameter_scalings() -> List[str]:
+        """Describes how to unscale the parameters - like time, timescale, flux or do not scale"""
```

Comment:
Could you please list all the possible variants here

Reply:
done